SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex

```diff
@@ -969,3 +972,102 @@ \subsection{CMakeList.txt}
 set(ITKVideoBridgeOpenCV_SYSTEM_INCLUDE_DIRS ${OpenCV_INCLUDE_DIRS})
 set(ITKVideoBridgeOpenCV_SYSTEM_LIBRARY_DIRS ${OpenCV_LIB_DIR})
 \end{minted}
+
+\section{Contributing with a Remote Module}
+\label{sec:ContributingRemoteModules}
+
+For most ITK community members, the modularization of the toolkit is relatively
+transparent. The default configuration includes all the (default) modules into
+the ITK library, which is used to build their own ITK applications.
+
+For ITK developers and code contributors, the modular structure imposes rules
+for organizing the source code, building the library and contributing to the
+ITK source code.
+
+A Module may be distributed outside the main ITK repository, but it may be
+built into ITK as a Remote Module once downloaded into the local copy of
+ITK source tree. The Remote Module infrastructure enables fast dissemination of
+research code through ITK without increasing the size of the main repository.
+The Insight Journal (\url{http://www.insight-journal.org/}) adds support for
+ITK module submissions with automatic dashboard testing (see
+Section~\ref{sec:CDash} on page~\pageref{sec:CDash} for further details.).
+
+The source code of a Remote Module can be downloaded by CMake (with a CMake
+variable switch) at ITK CMake configuration time, making it a convenient way to
+distribute modular source code.
+
+\subsection{Policy for Adding and Removing Remote Modules}
+\label{subsec:RemoteModuleAddRemovePolicy}
+
+A module can be added to the list of remotes if it satisfies the following
+criteria:
+\begin{itemize}
+\item There is a peer-reviewed article in an online, open access journal (such
+as the Insight Journal) describing the theory behind and usage of the module.
+\item There is a nightly build against ITK master on the CDash dashboard that
+builds and passes tests successfully.
+\item  A name and contact email exists for the dashboard build. The maintainer
+of the dashboard build does not necessarily need to be the original author of
+the Insight Journal article.
+\item The license should be compatible with the rest of the toolkit. That is
+it should be an OSI approved license without copyleft or non-commercial
+restrictions. Ideally, it should be an Apache 2.0 license assigned to the
+Insight Software Consortium as found in the rest of the toolkit.
```

Comment:
We should also mention here that the repository should not contain patented code/algorithms/methods.

Reply:
Done.